### DISCUSSION TITLE
max-content in grid-template-rows

### DISCUSSION THREAD

**Discussion by jaeyson** [tailwindlabs/tailwindcss#9111] [Ideas]
Hi!

I'd like to know if there's a possibility to add max-content in grid template rows (`grid-template-rows: max-content`)? That's all, thank you 🙏

**Comment by ichenglin** [comment]
You could use **[tailwind arbitrary values](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values)** to apply **native css** values,
so **in this case** it would be **`grid-rows-[max-content]`**.

**Reply by jaeyson** [reply]
Ah makes sense, thanks alot!
